core/src/wallet/cosmos/CosmosLikeMessage.cpp

```diff
@@ -272,6 +272,7 @@ rapidjson::Value CosmosLikeMessage::toJson(rapidjson::Document::AllocatorType& a
         addString(kProposalId, content.proposalId, jsonContent, allocator);
 
         // cosmos::MsgVote::option
+        // FIXME autogenerated to_string() returns incorrect case (returns UPPERCASE but should be CamelCase)
         auto option = api::to_string(content.option);
```

Comment:
You can probably fix that by using the static functions in `CosmosLikeConstants()`

Reply:
Good idea, will do.